docs/guides/MantaSigner.md

```diff
@@ -0,0 +1,212 @@
+# Manta Signer
+
+Manta Signer is a desktop application that protects your spending secrets and builds zero-knowledge proofs. It allows you to send and receive private (or "zk") assets.
+
+## Installation
+
+The latest version of Manta Signer can be installed from [here](https://signer.manta.network/).
+
+## Creating a New Account
+
+To create a new account, launch the Signer and select the first option `Create Account`.
+
+<br/>
+
+   <div style={{textAlign: 'center'}}>
+    <img alt="signer_setup_page" src="/img/guides/signer_setup_page.png" width="50%"/>
+   </div>
+<br/>
+
+Next you will have to pick a password that you will use to unlock the Signer. Please pick a strong password which is at least 8 characters long.
+
+<br/>
+
+   <div style={{textAlign: 'center'}}>
+    <img alt="signer-create-password-page" src="/img/guides/signer-create-password-page.png" width="50%"/>
+   </div>
+<br/>
+
+You will be brought to a page that displays your secret recovery phrase. Click the hidden icon in order to reveal it.
+
+<br/>
+
+   <div style={{textAlign: 'center'}}>
+    <img alt="signer-recovery-phrase-hidden" src="/img/guides/signer-recovery-phrase-hidden.png" width="50%"/>
+   </div>
+<br/>
+
+Please write down your recovery phrase offline, somewhere hidden. It is the only way to recover your assets if you lose access to your computer or forget your password! If anyone learns this phrase, they can steal all of your private assets.
```

Comment:
:::warning
Write down your recovery phrase offline, somewhere hidden and secure. It is the only way to recover your assets if you lose access to your computer or forget your password!
:::